spec/controllers/hyrax/dashboard/collections_controller_spec.rb

```diff
@@ -294,6 +294,47 @@
 
         expect(CollectionBrandingInfo.where(collection_id: collection.id, role: "logo", alt_text: "Logo alt Text", target_url: "http://abc.com").where("local_path LIKE '%logo.gif'")).to exist
       end
+
+      it "doesn't save linkurls with embedded html" do
+        val = double(["/public/logo.gif"])
+        allow(val).to receive(:file_url).and_return("/public/logo.gif")
+        allow(Hyrax::UploadedFile).to receive(:find).with("1").and_return(val)
+
+        allow(File).to receive(:split).with(any_args).and_return(["logo.gif"])
+        allow(FileUtils).to receive(:cp).with(any_args).and_return(nil)
+
+        put :update, params: { id: collection, logo_files: [1], alttext: ["Logo alt Text"], linkurl: ["<script>remove_me</script>"], collection: { creator: ['Emily'] }, update_collection: true }
+        collection.reload
+
+        expect(
+          CollectionBrandingInfo.where(
+            collection_id: collection.id,
+            role: "logo",
+            alt_text: "Logo alt Text",
+            target_url: "<script>some content</script>"
```

Comment:
Should this be `<script>remove_me</script>`?

Reply:
Yes, sorry. I fixed that in the latest push.